clusters/build-clusters/01_cluster/openshift-image-registry/cluster_config.yaml

```diff
@@ -4,7 +4,13 @@ kind: Config
 metadata:
   name: cluster
 spec:
+  nodeSelector:
+    node-role.kubernetes.io/infra: ""
```

Comment:
Nit: Can we use something that does not have `kubernetes.io` in the name, as this label is not official at all and specific to what we do `dptp.openshift.io` or sth)?

Reply:
I guess that the name make it shows up when we do

```
oc --context build01 get node ip-10-0-134-111.ec2.internal
NAME                           STATUS   ROLES          AGE    VERSION
ip-10-0-134-111.ec2.internal   Ready    infra,worker   155m   v1.19.0+7070803
```

I feel it is a nice-to-have thing.